examples/index.html

```diff
@@ -80,7 +80,9 @@
 
       // create an image
       img = L.distortableImageOverlay("example.png", {
-        suppressToolbar: false,
+        // defaultSelector can be different handles like "lock", "rotate", or "distort"
+        // default set to "rotate"
+        defaultSelector: '',
```

Comment:
>I started reviewing this but I started to realize shouldn't we just implement the mode option to allow us to set any initial handles we want?

@sashadev-sky The user can select between these choices, as their initial handles. Did you mean something else?

Reply:
oh cool! Would we want to call this `defaultTool` then? This is so rad!

Reply:
@jywarren Done!

Reply:
What I mean is that we have the ability right now to pass in `mode: lock` as an option and the image will come out as locked initially. Wouldn't it make more sense to extend the functionality of the `mode` option to let you chose any handle to start with, with distort still being the default if you don't specify anything? Like `mode: rotate` should make the handles initially rotate handles? 

The one it starts out on should still be distort really. But we can show in documentation how to make it start on another tool. And perhaps we would want to make one for `Scale` as well and start chipping away at this hardcoding situation we've been using.

Reply:
So what I mean is that we shouldn't create a new option at all but use the one we already have.